lisa/features/hibernation.py

```diff
@@ -1,12 +1,28 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
+from typing import Any, cast
 
+from lisa import schema
 from lisa.feature import Feature
 
 FEATURE_NAME_HIBERNATION = "Hibernation"
 
 
+class HibernationSettings(schema.FeatureSettings):
+    type: str = FEATURE_NAME_HIBERNATION
+    install_by_platform: bool = True
```

Comment:
Because we don't support other installation in code for now, it doesn't need to provide the `install_by_platform` flag. Other than that, it should add `enabled`, and set to false by default. So even a vm size support hibernation, if it's not specified, it won't be added.